### PR TITLE
Adding new workshop, removing old

### PR DESCRIPTION
**Pull request by jimbobbennett**

### CODE REVIEW

**Comment by shanamatthews** [comment]
@GeekTrainer - assuming you want to approve this?

File: README.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ You can get started by creating an account on Azure. There's three different opt
...
 - [az hack create](./az-hack.md) is an [Azure CLI](https://docs.microsoft.com/cli/azure/?view=azure-cli-latest) [extension](https://docs.microsoft.com/cli/azure/azure-cli-extensions-overview?view=azure-cli-latest) which can be used to automatically provision several Azure resources, including a web server, database and artificial intelligence. Upon successful creation, [Git deployment](https://docs.microsoft.com/azure/app-service/deploy-local-git) will be enabled, allowing you to deploy your application using tools you're already familiar with.
+- [Hackathon starter templates](https://aka.ms/HackTemplates) is a Gist containing a list of hackathon starter template repos that each contain a basic implementation of an app that you can deploy to Azure and wire up to services such as [Cognitive Services](https://azure.microsoft.com/services/cognitive-services/?WT.mc_id=hackwithazure-github-jabenn). You can use these templates to create a new repo, then follow the instructions in the relevant README.md to configure the services and deploy the app.
```
[the remark this answers is not in the document]
Comment (jimbobbennett) [reply README.md]: Yup - will do

File: workshops/web-ai-happy-sad-angry/README.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
+# Happy, Sad, Angry Workshop
+
+This workshop shows how to build a web based game called **Happy, Sad, Angry** using [Python](https://www.python.org) and [Flask](http://flask.pocoo.org) along with some HTML and JavaScript running on [Microsoft Azure](https://azure.microsoft.com/free/students/?WT.mc_id=hackwithazure-github-jabenn). This game selects an emotion, and you have to try your best to show that emotion on your face. Once you have your best emotion face on, you take a picture with your camera and the web game will check to see what emotion you are showing using the [Azure Face API](https://azure.microsoft.com/services/cognitive-services/face/?WT.mc_id=hackwithazure-github-jabenn). If it is the same as the one you were asked to show, you win.
```
[the remark this answers is not in the document]
Comment (jimbobbennett) [reply workshops/web-ai-happy-sad-angry/README.md]: Changed

File: workshops/web-ai-happy-sad-angry/code/app.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
+import random, os, io, base64
+from flask import Flask, render_template, request, jsonify
+from azure.cognitiveservices.vision.face import FaceClient
+from msrest.authentication import CognitiveServicesCredentials
+
+credentials = CognitiveServicesCredentials(os.environ['face_api_key'])
+face_client = FaceClient(os.environ['face_api_endpoint'], credentials=credentials)
+
+emotions = ['anger','contempt','disgust','fear','happiness','sadness','surprise']
```
[the remark this answers is not in the document]
Comment (jimbobbennett) [reply workshops/web-ai-happy-sad-angry/code/app.py]: Makes sense!

File: workshops/web-ai-happy-sad-angry/steps/CheckTheEmotion.md [hunk 4]
```diff
@@ -0,0 +1,337 @@
...
+
+  ```python
+  azure-cognitiveservices-vision-face
+  ```
+
+* Save the file
+
+* Install the new package from the terminal using the following command:
+  
+  ```sh
```
[the remark this answers is not in the document]
Comment (jimbobbennett) [reply workshops/web-ai-happy-sad-angry/steps/CheckTheEmotion.md]: Changed all to `pip`. Seeing as Python 2 is out of support, I guess we can assume they'll be on python3

File: workshops/web-ai-happy-sad-angry/steps/CheckTheEmotion.md [hunk 5]
```diff
@@ -0,0 +1,337 @@
...
+
+* Add the following to the bottom of the file:
+
+  ```python
+  azure-cognitiveservices-vision-face
+  ```
+
+* Save the file
+
+* Install the new package from the terminal using the following command:
```
[the remark this answers is not in the document]
Comment (jimbobbennett) [reply workshops/web-ai-happy-sad-angry/steps/CheckTheEmotion.md]: Nope, not in this workshop as we'd have to explain the concepts and teach how to use them. Would be better to link to a separate doc/workshop if they want to use them.

Just thinking about the cognitive load, especially for someone new to Python

[the remark this answers is not in the document]
Comment (GeekTrainer) [reply workshops/web-ai-happy-sad-angry/steps/CheckTheEmotion.md]: I'll go back in and update it :-) We should be teaching them best practices. I'm also not a fan of asking them to install a library globally. I've delivered this demo before using virtual environments and it went off without a hitch.

**Comment by jimbobbennett** [comment]
@GeekTrainer - I've added virtual env stuff.

**Comment by GeekTrainer** [comment]
🚢

**Pull request merged**